share/wake/lib/system/remote_cache_api.wake

```diff
@@ -1084,3 +1142,31 @@ def mkCacheSearchResponse (json: JValue): Result CacheSearchResponse Error =
     obytes
     | Match
     | Pass
+
+def hashUsage =
+    Usage 0 0.0 0.1 0 0 0
+
+# RSC version of computing hashes for files, similar to Hashcode target in path.wake
+def rsciHashFile (file: String): Result String Error =
```

Comment:
Is the `i` meaningful or a typo?  And you say "similar to", but how do they differ?  As on the meeting notes, any way to merge the two?